### DISCUSSION TITLE
fatal: No url found for submodule path 'ESMExtreme.github.io' in .gitmodules<title>

### DISCUSSION THREAD

**Discussion by ESMExtreme** [community/community#41809] [Actions]
### Select Topic Area

Product Feedback

### Body

Hello
when i try to update my test website it shows:
fatal: No url found for submodule path 'ESMExtreme.github.io' in .gitmodules
![image](https://user-images.githubusercontent.com/56802707/207816470-15843348-a513-4eca-b684-885ede90a65b.png)

**Comment by jfullstackdev** [comment]
as far as I can understand it, you have a GitHub Page, but the repo that is hosting it has a submodule,

it's a separate repo, that's what I can guess, if it has the GitHub Page URL you need to get that, if none, it's 404
